---
nav_msgs/Path raw_path